test/functional/tools/validation_image_metadata.xml

```diff
@@ -0,0 +1,154 @@
+<tool id="validation_image_metadata" name="validation_image_metadata" version="1.0">
+    <macros>
+        <!-- Tests are not executed if they have no output checks, thus define a dummy check -->
+        <xml name="test-output">
+            <output name="output">
+                <assert_contents>
+                    <has_size value="0" delta="0" />
+                </assert_contents>
+            </output>
+        </xml>
+   </macros>
+    <command><![CDATA[
+      cat /dev/null > testfile
+    ]]></command>
+    <inputs>
+        <!-- Checks for "axes" metadata -->
+        <param name="input_axes_yx" type="data" format="data" optional="true">
+            <validator type="dataset_metadata_equal" metadata_name="axes" value="YX" />
```

Comment:
Can you please also add one test with multiple validators?

Reply:
Not quite sure what you mean. There are the `input_multipage_tiff` and `input_unsupported_tiff_compression` tests for example, that have multiple validators. Do you mean using any specific validators?